akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -290,7 +301,9 @@ object Sink {
    * element is dropped and the stream continues.
    *
    * See also [[Flow.mapAsyncUnordered]]
+   * @deprecated Use `foreachAsync` instead, it allows you to choose your own execution context.
```

Comment:
I don't think that is needed, the Scala `@deprecated` annotation should be enough